packages/delisp-core/__tests__/readSyntax.ts

```diff
@@ -0,0 +1,125 @@
+import { readSyntax } from "../src/index";
```

Comment:
hm, I feel this module is a bit redundant with the tests for S-expression `reader.ts`, `compiler.ts` and `evaluate.ts`.

Does it add enough value? `compiler.ts` and `evaluate.ts` are going to be very very stable, but this new `readSyntax.ts` can be annoying as we change the syntax.

Reply:
Maybe a bit early indeed, plus I guess the only thing we need to verify is that `readSyntax === compose(convert, read)`, which is pretty trivial anyway :)